Task_1/FeTS_Challenge.ipynb

```diff
@@ -704,13 +704,68 @@
    "source": [
     "scores_dataframe"
    ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Produce NIfTI files for best model outputs on the validation set\n",
+    "Now we will produce model outputs to submit to the leader board.\n",
+    "\n",
+    "At the end of every experiment, the best model (according to average ET, TC, WT DICE) \n",
+    "is saved to disk at: ~/.local/workspace/checkpoint/\\<checkpoint folder\\>/best_model.pkl,\n",
+    "where \\<checkpoint folder\\> is the one printed to stdout during the start of the \n",
+    "experiment (look for the log entry: \"Created experiment folder experiment_##...\" above)."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from fets_challenge import model_outputs_to_disc\n",
+    "\n",
+    "# you will need to specify the correct experiment folder and the parent directory for\n",
+    "# the data you want to run inference over\n",
+    "checkpoint_folder='experiment_1'\n",
+    "data_path = '/raid/edwardsb/validation_set'\n",
+    "\n",
+    "# you can keep these the same if you wish\n",
+    "best_model_path = '/home/edwardsb/.local/workspace/checkpoint/' + checkpoint_folder + '/best_model.pkl'\n",
```

Comment:
For these paths, it might be worth doing something like this so the user doesn't need to modify their home directory:

```
from pathlib import Path
home = str(Path.home())
best_model_path = f'{home}/.local/workspace/checkpoint/{checkpoint_folder}/best_model.pkl'
outputs_path = f'{home}/.local/workspace/checkpoint/{checkpoint_folder}/model_outputs'
```

Reply:
Good catch Patrick, thanks. I missed that this was in the part I was suggesting did not need to be changed as well.